homeassistant/components/habitica/button.py

```diff
@@ -123,3 +343,8 @@ def available(self) -> bool:
         if self.entity_description.available_fn:
             return self.entity_description.available_fn(self.coordinator.data)
         return True
+
+    @property
+    def entity_picture(self) -> str | None:
+        """Return the entity picture to use in the frontend, if any."""
+        return self.entity_description.entity_picture
```

Comment:
Isn't this the default implementation?

Also, everything starts with ASSETSURL, might as well move that here so you can remove that prefix

Reply:
Is that better now? 😁